README.md

```diff
@@ -25,7 +25,7 @@ Note: this library was once called [jdbc-pg-sanity](https://clojars.org/jdbc-pg-
 
 ## Installation
 
-Add `mpg` as a leiningen dependency:
+Add `mpg` as a leiningen or boot dependency:
 
 ```clojure
 [mpg "1.2.0"]
```

Comment:
Nope. Shane will do this as part of the release. It'll probably go out as 1.3 with some notes about how we should now support inserts properly.